src/api/entities/Instruction/types.ts

```diff
@@ -5,8 +5,13 @@ import { EventIdentifier } from '~/types';
 
 export enum InstructionStatus {
   Pending = 'Pending',
-  Executed = 'Executed',
   Failed = 'Failed',
+  Rejected = 'Rejected',
+  Success = 'Success',
+  /**
+   * @deprecated v6.0 chain will specify Success or Rejected. "Executed" is kept for historical instructions
+   */
+  Executed = 'Executed',
```

Comment:
@prashantasdeveloper I kept this for when interacting with middleware. Not sure if we could completely remove it or not though

Reply:
We can remove this. Executed in Middleware should point as Success in SDK